common/pb/event_notify_user.proto

```diff
@@ -3,8 +3,10 @@ import "session_event.proto";
 message Event_NotifyUser {
 
     enum NotificationType {
+        UNKNOWN = 0;
```

Comment:
👍  though a comment referencing that the first enum value is what gets selected for non-optional fields the parser doesn't understand might be useful